code/game/objects/structures/fluff.dm

```diff
@@ -285,3 +285,43 @@
 /obj/structure/fluff/tram_rail/anchor
 	name = "tram rail anchor"
 	icon_state = "anchor"
+
+/obj/structure/fluff/desk_bell
+	name = "desk bell"
+	desc = "The cornerstone of any customer service job. You feel an unending urge to ring it."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "desk_bell"
+	pass_flags = PASSTABLE
+	layer = OBJ_LAYER
+	deconstructible = FALSE
+	anchored = FALSE
+	/// A cooldown timer so people don't pull out their autoclickers
+	var/fun_inhibitor = FALSE
```

Comment:
Use the cooldown defines

Also name it something useful

Reply:
forgot we had cooldown defines, give me  a bit and I'll fix it

Reply:
done